tests/TestCase/Console/ConsoleIoTest.php

```diff
@@ -280,7 +295,7 @@ public function testErr()
      * Tests abort() wrapper.
      *
      * @expectedException \Cake\Console\Exception\StopException
-     * @expectedExceptionMessage 1
+     * @expectedExceptionMessage Some error
```

Comment:
Can you move these annotations to `expectedException*()` calls please as I have done in 4.x to avoid merge conflicts.